apps/internal/oauth/oauth.go

```diff
@@ -107,6 +112,9 @@ func (t *Client) Credential(ctx context.Context, authParams authority.AuthParams
 		}
 		tr, err := cred.TokenProvider(ctx, params)
 		if err != nil {
+			if len(scopes) == 0 {
+				return accesstokens.TokenResponse{}, fmt.Errorf("token request had an empty authority.AuthParams.Scopes, which may cause the following error: %w", err)
```

Comment:
Should we fmt the err before returning an error that uses the format?
  see line 207 below

Reply:
You mean, to shorten this line?

Reply:
Consistency and coding patterns was the feedback.  We inline the format in some cases and create a temp to immediately return in others.  No difference in the end result but it does at a glance imply something is different